code/game/objects/structures/musician.dm

```diff
@@ -60,8 +60,11 @@
 	for(var/mob/M in get_hearers_in_view(15, source))
 		if(!M.client)
 			continue
-		if(M.client.prefs.hear_instruments)
-			M.playsound_local(source, soundfile, 100, falloff = 5)
+		if(!M.client.prefs.hear_instruments)
+			continue
```

Comment:
This causes people with instruments disabled to not be affected by the shakashuri

Reply:
good catch